config/application.rb

```diff
@@ -68,5 +68,12 @@ class Application < Rails::Application
 
     config.exceptions_app = routes
     config.active_job.queue_adapter = :sidekiq
+
+    config.rails_semantic_logger.format = :json
+    config.semantic_logger.add_appender(
+      io: $stdout,
+      level: config.log_level,
+      formatter: config.rails_semantic_logger.format,
+    )
```

Comment:
This one is only to test the review app since the review apps are not `Rails.env.production?`.

Reply:
Ignore. Review environment requires the production config.